src/parser/LexerConfig.ts

```diff
@@ -95,7 +95,7 @@ export const ErrorLiteral = createToken({name: 'ErrorLiteral', pattern: /#[A-Za-
 /* skipping whitespaces */
 export const WhiteSpace = createToken({
   name: 'WhiteSpace',
-  pattern: /[ \t\n\r]+/,
+  pattern: /\s+/,
```

Comment:
@sequba wrote in the OP:

> The standard does not mention whitespace characters other than SPACE, TAB, LINE FEED, and CARRIAGE RETURN so I assume it is up to us how we want to handle them.

Our implementation will become non-standard if we do whatever we want. I think we should make the behavior configurable, as we did in the past with other ideas that go beyond the ODFF standard: https://github.com/handsontable/hyperformula/issues/58

Reply:
IMO it is highly unlikely that someone wants to display `#ERROR!` for formulas including non-breakable spaces but it won't hurt to add a config option for controlling that behaviour. I'll work it.

Reply:
I added `allowAllWhitespace` parameter